### DISCUSSION TITLE
Why do I sometimes prefetch when I'm not using nuxtLink? How can I disable it?

### DISCUSSION THREAD

**Discussion by bbhxwl** [nuxt/nuxt#25987] [Questions]
I found that sometimes when nuxtLink is not used, it also prefetch, which is a waste of performance.

Is there a way to temporarily disable a page prefetch?

**Comment by hamedniroomand** [comment]
@bbhxwl See https://nuxt.com/docs/api/components/nuxt-link#nuxtlink

You can use the no-prefetch prop on nuxt-link to disable this behavior.

**Comment by bbhxwl** [comment]
> @bbhxwl See https://nuxt.com/docs/api/components/nuxt-link#nuxtlink
> 
> You can use the no-prefetch prop on nuxt-link to disable this behavior.

I have seen the one you posted, but I did not use any nuxtLink component and did not use it anywhere in the entire project.

**Comment by danielroe** [comment]
Prefetches can occur because:
 - resources are marked as needing prefetch in your build manifest - you can use the `build:manifest` hook to customise this on a per-file basis
 - NuxtLink prefetches resources for pages linked to when the link comes into the view port (this is not your issue, from what you've said)
 - ...and vite may prefetch some resources

**Reply by bbhxwl** [reply]
It seems difficult to control unexpected prefetches
